dotcom-rendering/src/web/lib/renderElement.tsx

```diff
@@ -696,24 +696,26 @@ export const renderElement = ({
 		case 'model.dotcomrendering.pageElements.YoutubeBlockElement':
 			return [
 				true,
-				<YoutubeBlockComponent
-					format={format}
-					key={index}
-					hideCaption={hideCaption}
-					// eslint-disable-next-line jsx-a11y/aria-role
-					role="inline"
-					adTargeting={adTargeting}
-					isMainMedia={isMainMedia}
-					id={element.id}
-					assetId={element.assetId}
-					expired={element.expired}
-					overrideImage={element.overrideImage}
-					posterImage={element.posterImage}
-					duration={element.duration}
-					mediaTitle={element.mediaTitle}
-					altText={element.altText}
-					origin={host}
-				/>,
+				<Island>
```

Comment:
There is deliberately no `deferUntil` here in order to force hydration as soon as possible. This will reduce the likelihood of users clicking to play a video before hydration has completed.